test/integration/production/test/security.js

```diff
@@ -94,6 +94,25 @@ module.exports = (context) => {
       }
     })
 
+    it('should not throw a server error for non-existent traversing paths', async () => {
+      const pathsToCheck = [
+        `/_next/static/../../server.php`,
+        `/public/plugins/../../server.php`,
+        `/example-page/../../../does-not-exist`,
+        `/public/../../../../../../conf/defaults.ini`,
+      ]
+      for (const path of pathsToCheck) {
+        const res = await fetchViaHTTP(context.appPort, path)
+        try {
+          expect([400, 404].includes(res.status)).toBeTruthy()
```

Comment:
Since the above test is handling the 404 case, can this one focus on the `400`s cases instead (I assume that means leaving off the first path).

Reply:
So it appears as though they are `404`s. I wonder if this test is necessary (code owners can decide). It's the same assertion, but testing against different paths and describing a different assumption.

Reply:
It appears as though I instead can change the test in `test/unit/isolated/require-page.test.ts`